src/wire/sixlowpan.rs

```diff
@@ -51,13 +51,271 @@ impl<'a> Address<'a> {
 
                 Ok(ipv6::Address::from_bytes(&bytes))
             }
-            Address::WithContext(_) => Err(Error::NotSupported),
+            Address::WithContext(ctx) => {
+                if ctx.is_empty() {
+                    let mut bytes = [0; 16];
+                    bytes[0] = 0xfe;
```

Comment:
Could we have constants for these, a note about them, or a mention of the relevant RFC section?